libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -567,6 +566,13 @@ private void initializeMapPaddingAdjustor(MapView mapView, MapboxMap mapboxMap)
     mapPaddingAdjustor = new MapPaddingAdjustor(mapView, mapboxMap);
   }
 
+  private void initializeMapSymbolManager(MapView mapView, MapboxMap mapboxMap) {
+    Bitmap markerBitmap = ThemeSwitcher.retrieveThemeMapMarker(mapView.getContext());
```

Comment:
They could, but this would also make it harder to test - increasing the amount of mocking work we would need to do with `Context` and the current theme.